mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/config/MasterConfiguration.java

```diff
@@ -360,6 +360,10 @@ default Duration getSchedulerIntervalBetweenRetries() {
     @Default("60000") // 1 minute
     int getAssignmentIntervalInMs();
 
+    @Config("mantis.job.costsCalculator.class")
+    @Default("io.mantisrx.master.jobcluster.job.CostsCalculator")
```

Comment:
Change this default value.

Reply:
Should this value be "CostsCalculatorFactory"? All these layers of abstraction make it really confusing. I'm not seeing the value. Can we just have a CostsCalculatorFactory interface and ask for an impl class in the config?

Reply:
Fair enough. I'll get rid of it. I was hoping to have some structure around how we initialize extendable components. The other diff (https://github.com/Netflix/mantis/pull/319/files) uses this idea/interface for a number of components (search for MantisExtension). But if you think having this defined in documentation/code is better than having an explicit interface, we can eliminate it.